charts/platform/values.yaml

```diff
@@ -44,17 +49,12 @@ scheduler:
 
 # -- Deployment resources
 resources:
-  {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  limits:
+    cpu: 100m
+    memory: 256Mi
+  requests:
+    cpu: 100m
+    memory: 256Mi
```

Comment:
Setting some initial values, we may need to modify these as time goes on - or once we know better, we can comment them out and have them as suggested values, inviting the user to pick their own.